conf/application.conf

```diff
@@ -273,7 +273,7 @@ play.filters {
     #permittedCrossDomainPolicies = "master-only"
 
     # The Content-Security-Policy header. If null, the header is not set.
-    contentSecurityPolicy = "default-src 'self'; script-src https://code.jquery.com https://cdnjs.cloudflare.com 'self'; style-src https://fonts.googleapis.com 'self'; font-src https://fonts.gstatic.com"
+    contentSecurityPolicy = "default-src 'self'; script-src https://code.jquery.com https://cdnjs.cloudflare.com 'self' ; style-src https://fonts.googleapis.com https://cdn.jsdelivr.net 'self' 'unsafe-inline'; font-src https://fonts.gstatic.com https://cdn.jsdelivr.net 'self' 'unsafe-inline'; connect-src https://delphi.cs.uni-paderborn.de/api-legacy/features 'self' 'unsafe-inline';"
```

Comment:
Please test if you can remove the `delphi.cs...` link here. I think you don't need that anymore.

Reply:
After removing hard-coded url in frontend, links from this CS policy has been removed. Thanks!

Reply:
Great! Thanks!